src/py/mat3ra/made/tools/utils.py

```diff
@@ -96,3 +96,100 @@ def get_norm(vector: List[float]) -> float:
         float: The norm of the vector.
     """
     return float(np.linalg.norm(vector))
+
+
+# Condition functions:
+# TODO: Predefined equations can be exported using a factory or enum
+def _is_2d_point_in_circle(coordinate: List[float], x=0, y=0, r=1) -> bool:
```

Comment:
the is_* functions should act on a coordinate. If you want to have an internal helper function for a 2d point, it can be done local to this file. We can also move all these to utils later

Reply:
Isn't this one local?